internal/broker/broker.go

```diff
@@ -204,10 +204,24 @@ func (b *Broker) GetAuthenticationModes(sessionID string, supportedUILayouts []m
 	_, err = os.Stat(session.cachePath)
 	tokenExists := err == nil
 
+	// Checks if the provider is accessible and if device authentication is supported
+	providerReachable := true
+	r, err := http.Get(strings.TrimSuffix(b.auth.providerURL, "/") + "/.well-known/openid-configuration")
```

Comment:
double // doesn’t work or this is just for principles? :)

Reply:
To be honest, I just used the same code that the `oidc` package uses :joy: but I'm pretty sure that double slashes invalidate the URI.